### PR TITLE
DOC Add doc in `CommandRegistry` about when it is updated

### PR DESCRIPTION
**Pull request by lucyleeow**
Adds some info about when `CommandRegistry` is updated. Uses info learned when debugging https://github.com/napari/npe2/issues/314

Happy to change wording or move to a better place.

### CODE REVIEW

**Comment by lucyleeow** [comment]
cc @DragaDoncila

**Comment by lucyleeow** [comment]
I don't think the CI failures are related to this doc change.

File: src/npe2/_command_registry.py [hunk 1]
```diff
@@ -39,6 +39,13 @@ def resolve(self) -> Callable:
 
 
 class CommandRegistry:
+    """Registry of commands, updated when `PluginManager.activate` run.
```
Comment (DragaDoncila) [inline src/npe2/_command_registry.py:42]:
```suggestion
    """Registry of commands, updated on `PluginManager.activate`.
```

File: src/npe2/_command_registry.py [hunk 2]
```diff
@@ -39,6 +39,13 @@ def resolve(self) -> Callable:
...
+
+    `PluginManager.activate` is only run on `CommandRegistry.get` (when we get the
```
Comment (DragaDoncila) [inline src/npe2/_command_registry.py:44]:
```suggestion
    `PluginManager.activate` is only run on `CommandRegistry.get` (when we resolve the
```

**Review by DragaDoncila — APPROVED** [review]
Small nits, thanks @lucyleeow 🙏

**Comment by DragaDoncila** [comment]
Looks like our codecov token might have expired, or something:

```sh
[2023-10-10T02:14:55.728Z] ['error'] There was an error running the uploader: Error uploading to [https://codecov.io:]
(https://codecov.io/) Error: There was an error fetching the storage URL during POST: 400 - [ErrorDetail(string='This 
repository has been deactivated. To resume uploading to it, please activate the repository in the codecov UI: 
https://app.codecov.io/github/napari/npe2/settings', code='invalid')]

```

 @Czaki do you know what I'd need to do to fix this?

**Comment by lucyleeow** [comment]
Thanks changes made!

**Pull request merged**